include/mbedtls/cipher.h

```diff
@@ -228,7 +228,8 @@ typedef struct {
      */
     unsigned int iv_size;
 
-    /** Flags to set. For example, if the cipher supports variable IV sizes or variable key sizes. */
+    /** Flags to set. For example, if the cipher
```

Comment:
Suggestion: Bitflag comprised of `MBEDTLS_CIPHER_VARIABLE_IV_LEN` and `MBEDTLS_CIPHER_VARIABLE_KEY_LEN` indicating whether the cipher supports variable IV or variable key sizes, respectively.`

Reply:
done